package.json

```diff
@@ -63,22 +63,23 @@
     "npm": ">= 6.0.0"
   },
   "dependencies": {
-    "@energyweb/credential-governance": "^1.0.1-alpha.68.0",
+    "@energyweb/credential-governance": "^1.0.1-alpha.127.0",
     "@energyweb/ekc": "^0.6.6",
-    "@energyweb/onchain-claims": "^1.0.1-alpha.68.0",
+    "@energyweb/onchain-claims": "^1.0.1-alpha.127.0",
     "@energyweb/staking-pool": "^1.0.0-rc.14",
+    "@energyweb/vc-verification": "^1.0.1-alpha.127.0",
     "@ensdomains/ens": "^0.6.2",
-    "@ew-did-registry/claims": "^0.6.3-alpha.596.0",
-    "@ew-did-registry/credentials-interface": "^0.6.3-alpha.596.0",
-    "@ew-did-registry/did": "^0.6.3-alpha.596.0",
-    "@ew-did-registry/did-document": "^0.6.3-alpha.596.0",
-    "@ew-did-registry/did-ethr-resolver": "^0.6.3-alpha.596.0",
-    "@ew-did-registry/did-ipfs-store": "^0.6.3-alpha.596.0",
-    "@ew-did-registry/did-resolver-interface": "^0.6.3-alpha.596.0",
-    "@ew-did-registry/jwt": "^0.6.3-alpha.596.0",
-    "@ew-did-registry/keys": "^0.6.3-alpha.596.0",
-    "@ew-did-registry/proxyidentity": "^0.6.3-alpha.596.0",
-    "@ew-did-registry/revocation": "^0.6.3-alpha.596.0",
+    "@ew-did-registry/claims": "^0.6.3-alpha.626.0",
+    "@ew-did-registry/credentials-interface": "^0.6.3-alpha.626.0",
+    "@ew-did-registry/did": "^0.6.3-alpha.626.0",
+    "@ew-did-registry/did-document": "^0.6.3-alpha.626.0",
+    "@ew-did-registry/did-ethr-resolver": "^0.6.3-alpha.626.0",
+    "@ew-did-registry/did-ipfs-store": "^0.6.3-alpha.626.0",
```

Comment:
@JGiter @nichonien I updated all ew-did-registry and ew-credentials to latest versions